### PR TITLE
Update bazel to 0.20 and all dependencies as a result

### PR DESCRIPTION
**Pull request by jverkoey**
Kokoro's Bazel version has been increased from 0.11.0 to 0.20.0

As part of this change, the `.kokoro` bazel header rewrite phase's rewrites for MDF libraries has been made less generic because MDFInternationalization can now be imported using framework-style imports as expected. MDFTextAccessibility does not yet have this functionality, so we still need to rewrite it.

Version changes in the bazel workspace:

- bazelbuild/rules_apple has been increased from 0.3.0 to 0.9.0
- bazel_skylib has been increased from 0.2.0 to 0.6.0
- bazelbuild/rules_swift has been added at 0.4.0
- xctestrunner has been increased from 0.2.1 to 0.2.5
- material_internationalization_ios has been increased from v1.0.4 to v2.0.1
- material_text_accessibility_ios has been increased from fd570d71ae0124c75ad5af00e6b8b4b1668d5e40 to v2.0.0
- motion_interchange_objc has been increased from v1.6.0 to v2.0.0
- motion_animator_objc has been increased from v2.5.0 to v3.0.0
- motion_transitioning_objc has been increased from v5.0.0 to v6.0.0
- Swift version has been pinned to 3 for all swift libraries.

buildifier was ran on all affected build targets.

This unblocks https://github.com/material-components/material-components-ios/pull/5550.

### CODE REVIEW

**Comment by jverkoey** [comment]
This is ready for review.

File: .kokoro [hunk 1]
```diff
@@ -55,7 +55,7 @@ fix_bazel_imports() {
   rewrite_tests "s/import MaterialComponentsAlpha.Material(.+)/import components_\1_\1 \/\/ Alpha/"
   rewrite_tests "s/import MaterialComponents.Material(.+)/import components_\1_\1/"
   rewrite_source "s/import <Motion(.+)\/Motion.+\.h>/import \"Motion\1.h\"/"
-  rewrite_source "s/import <MDF(.+)\/MDF.+\.h>/import \"MDF\1.h\"/"
+  rewrite_source "s/import <MDFTextAccessibility\/MDFTextAccessibility\.h>/import \"MDFTextAccessibility.h\"/"
```
Comment (romoore) [inline .kokoro:58]: Looks like you're updating the `.kokoro` script as well. Would you please add a description of why that change is needed to the PR summary?

File: components/ActionSheet/BUILD [hunk 2]
```diff
@@ -51,11 +53,11 @@ mdc_objc_library(
     hdrs = native.glob(["src/TypographyThemer/*.h"]),
     includes = ["src/TypographyThemer"],
     sdk_frameworks = ["UIKit"],
+    visibility = ["//visibility:public"],
```
Comment (romoore) [inline components/ActionSheet/BUILD]: Tip: Next time please put `buildifier` changes in a precursor PR. That will help reduce "noise" for reviewers.

**Review by romoore — APPROVED** [review]

File: .kokoro (hunk 1 above)
Comment (jverkoey) [reply .kokoro:58]: Done.

**Pull request merged**